Makefile.common

```diff
@@ -27,6 +27,8 @@ MKDIR=mkdir -p
 EMPTY :=
 # $(SPACE) contains a single space
 SPACE := $(EMPTY) $(EMPTY)
+# $( ) suppresses warning from the alignments in the V_ macros below
+$(SPACE) :=
```

Comment:
This is wizard-level Makefile hacking :)

Reply:
It's also possible to do it by having, e.g.

```make
V_CC         = @$(info $(SPACE) CC $@)
```

but that looks a little odd. More importantly, it's relatively easy to find `$ ` recommended as a way of escaping a space, but rather harder to find the trick necessary to suppress the undefined variable warning, so it seems worth having the slightly strange trick above. Incidentally, that macro is principally used for escaping newlines without adding space (by writing `$\` instead of `\` at the end of the line.)